ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -12,14 +12,22 @@ import useNetworkPolicySimulator from './hooks/useNetworkPolicySimulator';
 import SimulationFrame from './simulation/SimulationFrame';
 import TopologyComponent from './TopologyComponent';
 import { EdgeState } from './components/EdgeStateSelect';
+import { NetworkScopeHierarchy } from './types/networkScopeHierarchy';
 
 export type NetworkGraphProps = {
     model: CustomModel;
     simulation: Simulation;
     selectedNode?: CustomNodeModel;
     edgeState: EdgeState;
+    scopeHierarchy: NetworkScopeHierarchy;
 };
-function NetworkGraph({ model, simulation, selectedNode, edgeState }: NetworkGraphProps) {
+function NetworkGraph({
+    model,
+    simulation,
+    selectedNode,
+    edgeState,
+    scopeHierarchy,
```

Comment:
I was going to mention if the prop drilling being a byproduct of this change is fine, but I think it's ok considering how a scope hierarchy change will result in many components having to update anyways and it's not that bad

Reply:
I was almost tempted to put this into a `Context` since the prop drilling crosses 4 components, but resisted since that hasn't been the pattern in the rest of the app from what I can tell. If we end up spreading the use of this to other places in the NG it might be worth it though.